### ISSUE TITLE
Correctly use Releases/Tags

### ISSUE THREAD

**Issue by tr7zw** [oobabooga/text-generation-webui#2659]
**Description**

Related to #2141.
After changes to the installers, create a new tag with the new files maybe with some info on what changed compared to the last version. Currently updates just get edited into the https://github.com/oobabooga/text-generation-webui/releases/tag/installers release, which is not how tags/releases are supposed to be used. Best-case this should be used in combination with #2141 to tag together a stable installer+webui codebase. The current setup makes it seem like there hasn't been any progress or fixes since Feb 9.

**Comment by ashleykleynhans** [comment]
Agreed, this project definitely needs tags and releases because it is causing massive headaches when things are pushed to the main branch and then everything becomes broken.
